### PR TITLE
[HOTFIX] remove this useless assignment

### PR DESCRIPTION
**Pull request by lamber-ken**
Be sure to do all of the following checklist to help us incorporate 
your contribution quickly and easily:

 - [x] Any interfaces changed? NO
 
 - [x] Any backward compatibility impacted? NO
 
 - [x] Document update required? NO

 - [x] Testing done YES
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - How it is tested? Please attach test report.
        - Is it a performance related change? Please attach the performance test report.
        - Any additional information to help reviewers in testing this change.
       
 - [x] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA. OK

### CODE REVIEW

**Comment by lamber-ken** [comment]
`status` was already assigned value `false`

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/1910/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.2, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/10164/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/2121/

File: core/src/main/java/org/apache/carbondata/core/locks/LocalFileLock.java [hunk 1]
```diff
@@ -112,7 +112,7 @@ public LocalFileLock(String lockFileLocation, String lockFile) {
         status = true;
       }
     } catch (IOException e) {
-      status = false;
+      // status = false;
```
Comment (qiuchenjian) [inline core/src/main/java/org/apache/carbondata/core/locks/LocalFileLock.java]: Should it print error message for IOException ?

Comment (lamber-ken) [reply core/src/main/java/org/apache/carbondata/core/locks/LocalFileLock.java]: it is no need

Comment (jackylk) [reply core/src/main/java/org/apache/carbondata/core/locks/LocalFileLock.java]: I suggest we print a warning message, and remove this assignment
@ravipesala @QiangCai

**Comment by lamber-ken** [comment]
hi, @jackylk, I think this may be part of code style as we talk about in wechat group. So need someone control  
 the code clean progress

File: core/src/main/java/org/apache/carbondata/core/locks/LocalFileLock.java (hunk 1 above)
Comment (jackylk) [reply core/src/main/java/org/apache/carbondata/core/locks/LocalFileLock.java]: Yes, I think HdfsFileLock has similar problem. Actually there is a PR #2878 trying to fix all exception code style problem, by @kevinjmh . I suggest we can have it handled in that PR. Then for this PR, I think you can remove the assignment directly, same for HdfsFileLock

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/1957/

File: core/src/main/java/org/apache/carbondata/core/locks/LocalFileLock.java (hunk 1 above)
Comment (lamber-ken) [reply core/src/main/java/org/apache/carbondata/core/locks/LocalFileLock.java]: > Yes, I think HdfsFileLock has similar problem. Actually there is a PR #2878 trying to fix all exception code style problem, by @kevinjmh . I suggest we can have it handled in that PR. Then for this PR, I think you can remove the assignment directly, same for HdfsFileLock

ok, I see

**Comment by lamber-ken** [comment]
@jackylk , to avoid conflict, I should close this PR and create a new PR when #2878 finished.

**Pull request closed without merging**